code/pizzabot.js

```diff
@@ -5,3 +5,38 @@ const pepperoni = "Pepperoni Pizza"
 const pizzaPrice = 80
 
 //Put your Javscript code here:
+
+console.log(pizzaPrice)
+
+alert(`Hey! Happy to serve your pizza. On our meny we have ${vegetarian}, ${hawaiian}, ${pepperoni}!`)
+  
+  const orderName = prompt("Enter the name of the pizza you want to order today.", "");
+  if (orderName === vegetarian || orderName === hawaiian || orderName === pepperoni)
+	{
+    var orderQuantity =  prompt(
```

Comment:
Try to use `const` or `let` before using `var` if possible!
```suggestion
    const orderQuantity =  prompt(
```